docs/csharp/toc.yml

```diff
@@ -317,23 +322,26 @@
       href: programming-guide/inside-a-program/identifier-names.md
     - name: C# Coding Conventions
       href: programming-guide/inside-a-program/coding-conventions.md
-  - name: Main() and Command-Line Arguments
+  - name: Main() and command-line arguments
     items:
     - name: Overview
+      displayName: main method
       href: programming-guide/main-and-command-args/index.md
     - name: Command-Line Arguments
       href: programming-guide/main-and-command-args/command-line-arguments.md
     - name: "How to display command-line arguments"
       href: programming-guide/main-and-command-args/how-to-display-command-line-arguments.md
     - name: Main() Return Values
       href: programming-guide/main-and-command-args/main-return-values.md
-  - name: Programming Concepts
+  - name: Programming concepts
     items:
     - name: Overview
+      displayName: programming concepts
```

Comment:
This is a really smart idea whenever we've added a node for the new TOC structure. Should we do the same in other areas? /cc @mairaw

Reply:
Yes, it's too bad the TOC filter doesn't pick up keywords in titles that don't have a doc linked to them.